.github/workflows/test.yml

```diff
@@ -98,7 +98,6 @@ jobs:
 
       - name: Run cargo fmt
         run: |
-          rustup component add rustfmt --toolchain nightly-2021-06-21-x86_64-unknown-linux-gnu
           cargo +nightly-2021-06-21 fmt --all -- --check
```

Comment:
if you remove the line that's removed, what nightly version of rustfmt does it install in this command (i.e. `-aarch64-apple-darwin` `-x86_64-apple-darwin`) or does the line that you've removed install `rustfmt` using `rustup`, whereas running `cargo fmt ...` uses the version of `cargo-fmt` that gets installed with `cargo install ...`?

Reply:
@ltfschoen I added it back. But its still not working. I reported an issue here https://github.com/paritytech/subport/issues/229. Something is wrong with polkadot 0.8.30.